psycopg/replication_cursor_type.c

```diff
@@ -124,8 +138,9 @@ consume_stream(replicationCursorObject *self,
     CLEARPGRES(curs->pgres);
 
     self->consuming = 1;
+    set_status_interval(self, keepalive_interval);
```

Comment:
It looks like this is going to overwrite the value provided in the function to start the stream.  I wonder if we could make `None` the new default value for this parameter and then if the client didn't specify it explicitly, the value from starting the stream will be kept.